autogen.sh

```diff
@@ -5,7 +5,31 @@ set -xe
 type autoreconf || exit 1
 type pkg-config || exit 1
 
-ctags_files=`make -f makefiles/list-translator-input.mak --no-print-directory`
+gen_bsdmakefile() {
+	set +xe
```

Comment:
why changing those shell options?  What is supposed to be the problem with `e` here, and why disabling `x`?